CONTRIBUTING.md

```diff
@@ -156,11 +157,47 @@ You can update your [fork](https://help.github.com/articles/about-forks/) of the
 GitHub has a [nice introduction](https://help.github.com/articles/github-flow/) to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions.
 
 ## Example pull request
-<img align="right" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
+<img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>  
 
-<br />
-<br />
 
+## Fixing Travis Remark errors
+
+We use a linter called [Remarkjs](https://github.com/remarkjs/remark-lint) to ensure all of our specification documentation is consistent.
+We have found this causes typical errors that are flagged by our continous integration service called [Travis CI](https://travis-ci.org/).
+This behavior is commonly seen when a PR is opened and Travis returns an error. 
+To fix this error please follow this process:
+
+#### 1. Download Remark-CLI and our style guide
+
+Remark-CLI can be downloaded with the help of [npm](https://www.npmjs.com/). 
+Once npm has been downloaded, please follow the [installation instructions on Remark's GitHub.](https://github.com/remarkjs/remark/tree/master/packages/remark-cli#install)
+
+We use a particular style guide for formatting. 
+To get this style guide please run: 
+
+```
+npm install remark-cli@5.0.0 remark-lint@6.0.2 remark-preset-lint-recommended@3.0.2 remark-preset-lint-markdown-style-guide@2.1.2
+```
+
+#### 2. Fix the flagged document
+
+Please go to the directory where the flagged file is and run remark like this:
+
+```
+remark flagged_file.md -o flagged_file_fixed.md
+```
+
+Please confirm this has fixed the file. 
+To do this, please run this:
+
+```
+remark flagged_file_fixed.md --frail
+```
+
+This command will return whether this file can pass Travis.
+If it passes, the `flagged_file_fixed.md` can update the `flagged_file.md`.
```

Comment:
Suggesting a code block doesn't work well, so just using straight markdown below:

----

If it passes, replace `flagged_file.md` with the contents of `flagged_file_fixed.md`,
`add` and `commit` the change:

```Shell
mv flagged_file_fixed.md flagged_file.md
git add flagged_file.md
git commit -m 'STY: Fixed Markdown style'
```